app/frontend/src/features/connections/useCancelFriendRequest.ts

```diff
@@ -1,32 +0,0 @@
-import { Empty } from "google-protobuf/google/protobuf/empty_pb";
```

Comment:
This was more of a file rename rather than deleted, as I realised I put this under `connections` rather `connections/friends`, but Git didn't seem smart enough to pick it up in this case after I added `reset` from the mutation to be returned here as well